supervisely/nn/benchmark/evaluation/coco/metric_provider.py

```diff
@@ -146,8 +148,8 @@ def json_metrics(self):
         ap_custom_by_class = dict(zip(self.cat_names, ap_custom_by_class))
         return {
             "mAP": base["mAP"],
-            "AP50": self.coco_metrics["AP50"],
-            "AP75": self.coco_metrics["AP75"],
+            "AP50": self.coco_metrics.get("AP50", "-"),
```

Comment:
None is better here. We can replace None to "-" later, in visualizer

Reply:
fixed